Client/Frontend/Browser/Toolbars/UrlBar/TopToolbarView.swift

```diff
@@ -379,6 +379,12 @@ class TopToolbarView: UIView, ToolbarProtocol {
     locationTextField.attributedPlaceholder = self.locationView.placeholder
     locationTextField.rightView = qrCodeButton
     locationTextField.rightViewMode = .never
+    
+    let dragInteraction = UIDragInteraction(delegate: self)
+    locationTextField.addInteraction(dragInteraction)
+
+    let dropInteraction = UIDropInteraction(delegate: self)
+    self.addInteraction(dropInteraction)
```

Comment:
Could we somehow pass this off to the BrowserViewController? It already has a drop interaction & delegate set up